scss/_tags.scss

```diff
@@ -0,0 +1,159 @@
+  --#{$prefix}tag-active-decoration-color: #{$tag-active-decoration-color};
+  --#{$prefix}tag-disabled-color: #{$tag-disabled-color};
+  --#{$prefix}tag-outline-color: var(--#{$prefix}link-hover-color);
+  // scss-docs-end tag-css-vars
+
+  display: inline-flex;
+  gap: var(--#{$prefix}tag-gap);
+  align-items: center;
+  padding: subtract(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5)) var(--#{$prefix}tag-padding-x) add(var(--#{$prefix}tag-padding-y), calc(var(--#{$prefix}tag-font-shift) * .5));
+  margin: var(--#{$prefix}tag-margin);
+  @include font-size(var(--#{$prefix}tag-font-size));
+  font-weight: var(--#{$prefix}tag-font-weight);
+  line-height: add(var(--#{$prefix}tag-font-size), .125rem);
+  color: inherit;
+  letter-spacing: var(--#{$prefix}tag-font-spacing);
+  background-color: transparent;
+  border: var(--#{$prefix}tag-border-width) solid var(--#{$prefix}tag-border-color);
+  outline-offset: 2px;
+  @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+
+  img,
+  svg {
+    width: var(--#{$prefix}tag-icon-size);
+    min-width: var(--#{$prefix}tag-icon-size); // Here to avoid weird behavior on wrap
+    height: var(--#{$prefix}tag-icon-size);
+    margin: add(calc(-.5 * var(--#{$prefix}tag-icon-size)), var(--#{$prefix}tag-font-shift)) 0 calc(-.5 * var(--#{$prefix}tag-icon-size)) var(--#{$prefix}tag-icon-margin-start);
+  }
+
+  .close {
+    position: relative;
+    width: var(--#{$prefix}tag-close-size);
+    min-width: var(--#{$prefix}tag-close-size); // Here to avoid weird behavior on wrap
+    height: var(--#{$prefix}tag-close-size);
+    margin: add(calc(-.5 * var(--#{$prefix}tag-close-size)), var(--#{$prefix}tag-font-shift)) var(--#{$prefix}tag-close-margin-end) calc(-.5 * var(--#{$prefix}tag-close-size)) var(--#{$prefix}tag-close-margin-start);
+    color: inherit;
+    background-color: transparent;
+    border: 0;
+    outline-offset: 2px;
+    @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+
+    &::before {
+      position: absolute;
+      top: calc(-1 * var(--#{$prefix}tag-padding-y));
+      right: calc(-1 * var(--#{$prefix}tag-padding-y));
+      bottom: calc(-1 * var(--#{$prefix}tag-padding-y));
+      left: calc(-1 * var(--#{$prefix}tag-padding-y));
+      content: "";
+      @include border-radius(var(--#{$prefix}tag-border-radius), var(--#{$prefix}tag-border-radius));
+    }
+
+    &::after {
+      display: block;
+      height: 100%;
+      content: "";
+      background-color: currentcolor;
+      mask-image: escape-svg(var(--#{$boosted-prefix}close-icon));
+      mask-repeat: no-repeat;
+      mask-position: 50%;
+      mask-size: subtract(var(--#{$prefix}tag-close-size), .5rem);
+    }
+
+    &:hover,
+    &:active {
+      color: var(--#{$prefix}tag-active-color);
+      background-color: var(--#{$prefix}tag-active-decoration-color);
+    }
+
+    &:focus[data-focus-visible-added] {
```

Comment:
Since it will be merged before the work on the focus. It'll need to be checked in this focus PR.